### ISSUE TITLE
Could canvasXpress no have a legend where there is no colour or fill aesthetic etc?

### ISSUE THREAD

**Issue by davidhodge931** [neuhausi/canvasXpress#154]
```
library(tidyverse)
library(palmerpenguins)
library(ggblanket)

p <- penguins |> 
  ggplot(aes(x = species, y = body_mass_g)) +
  geom_point() 

p

canvasXpress::canvasXpress(p)
```

**Comment by neuhausi** [comment]
Fixed in version 48.9
![cX-htmlwidget-1ede8ed07c1c5dac5989-cx (1)](https://github.com/neuhausi/canvasXpress/assets/254923/df522a16-6f73-4cda-9ead-fd81c8d5030d)

**Comment by davidhodge931** [comment]
Nice :)
